api/types/src/transaction.rs

```diff
@@ -347,6 +347,33 @@ pub struct SubmitTransactionRequest {
     pub signature: TransactionSignature,
 }
 
+#[derive(Debug, Serialize, Deserialize, Object)]
+pub struct SubmitTransactionsBatchExecutionResult {
```

Comment:
`SubmitTransactionsBatchExecutionResult` The name is misleading as this is not execution result - this is just the submission result. How aboue we use `BatchSubmitTransactionsResult` instead?